src/Decompiler/Structure/Region.cs

```diff
@@ -47,6 +47,17 @@ public class Region
         public List<AbsynStatement> Statements { get; set; }
         public Expression Expression { get; set; }
 
+        /// <summary>
+        /// Check if region contains only return statement
+        /// </summary>
```

Comment:
Thanks. I'd also change the comment to make it extra clear "Return true if region consists of a single AbsynReturn statement."

Reply:
Done. I have amended this commit one more time.